tests/markov/msm/test_tram.py

```diff
@@ -299,25 +315,25 @@ def harmonic(x0, x):
             bias = lambda x, x0=bias_center: harmonic(x0, x)
             bias_matrices[i, :, j] = bias(traj)
 
-    tram = TRAM(maxiter=100, connectivity='summed_count_matrix', save_convergence_info=True)
+    tram = TRAM(maxiter=100, connectivity='summed_count_matrix')
     assert tram.log_likelihood is None
 
-    tram.fit((dtrajs, bias_matrices))
+    model = tram.fit_fetch((dtrajs, bias_matrices))
 
     # energies are identical. so are count matrices. and transition matrices
-    assert np.allclose(tram.therm_state_energies, [0.15673362, 0.077853, 0.04456354, 0.05706922, 0.11557514])
-    assert np.allclose(tram.markov_state_energies, [1.0550639, 0.42797176])
-
-    model = tram.fetch_model()
-    assert np.allclose(model.stationary_distribution, [1.])
-    model.select(1)
-    assert np.allclose(tram.fetch_model().stationary_distribution, [0.3678024695571382, 0.6321975304428619])
-    assert np.allclose(tram.fetch_model().transition_matrix,
+    assert np.allclose(model.therm_state_energies, [0.15673362, 0.077853, 0.04456354, 0.05706922, 0.11557514])
+    assert np.allclose(model.markov_state_energies, [1.0550639, 0.42797176])
+
+    MEMM = model.markov_state_model_collection
```

Comment:
can we call this `msm_collection`? or something else entirely? from the name the purpose of the property isn't really clear